src/main/docker/scripts/export-file-based-secrets.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright 2017-2021 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Create a convenience function for info logging.
+info() {
+    echo "[$(date +%H:%M:%S.%3NZ) #$(printf '%03X\n' $$).??? INFO  -            -   ] ${0##*/}: $@"
+}
+
+# Create a convenience function for error logging.
+error() {
+    echo "[$(date +%H:%M:%S.%3NZ) #$(printf '%03X\n' $$).??? ERROR  -            -   ] ${0##*/}: $@"
+}
+
+# Create a function for exporting file-based secrets.
+#
+# For example, for each environment variable ending in the _FILE suffix:
+#
+#     ABC_PASSWORD_FILE=/var/somefile.txt
+#
+# read the contents of /var/somefile.txt (for example 'mypassword'), and export an environment variable named ABC_PASSWORD set to:
+#
+#     ABC_PASSWORD=mypassword
+export_file_based_secrets() {
+    env | while IFS= read -r env_var; do
+        local env_var_name=${env_var%%=*}
+        local env_var_value=${env_var#*=}
```

Comment:
This would be easily spoofed (by having an environment variable containing a newline).  You should instead use `env -0`:

```suggestion
    env -0 | while IFS='=' read -r -d '' env_var_name env_var_value; do
```